main.css

```diff
@@ -1,73 +1,81 @@
-#a-1 {
-    font-size: 15px;
-}
-
-#body {
+body {
     margin: 0;
     padding: 0;
     background-color: #eeeff2;
 }
 
-#btn-form {
-    font-family: Arial, Helvetica, sans-serif;
-    color: white;
-    font-size: 17px;
-    border: 3px;
-    border-bottom-color: black;
-    border-radius: 5px;
-    background: steelblue;
-    width: 100%;
-    height: 40px;
-}
+#sidebar {
+    /* float: right; */
```

Comment:
Hey thanks for the comments, could you be so kind to tall us how do you tested? because in our screen it seems very similar to the original, let me put you a screen,
![Screenshot from 2019-08-15 06-58-30](https://user-images.githubusercontent.com/2024897/63099593-3ded3c00-bf2a-11e9-9107-51a855b9099f.png)

Reply:
This looks better :+1:
However, I can still spot the bottom scrollbar.This means that the page is taking more than the screen width.
- Fix this issue, as it seem like you using 
- Check other required comments.
- Test it remotely using `raw.githack.com`, this way you can be sure anyone can view the page from any other computer.
Do this and submit another code review. :ok_hand:

Reply:
https://raw.githack.com/jcromerohdz/html-form/revert-1-signinForm/index.html

Reply:
Thanks